apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -249,23 +249,28 @@ export const EventSetupTab = (
                 isSearchable={false}
                 className="h-auto !min-h-[36px] text-sm"
                 options={multipleDurationOptions}
+                value={selectedMultipleDuration}
                 onChange={(options) => {
+                  let newOptions = [...options];
+                  newOptions = newOptions.sort((a, b) => {
+                    return a?.value - b?.value;
+                  });
```

Comment:
Is there a better way for this? Than doing double sorting.

Reply:
Yeah, I missed to change it. now we are sorting just once.